config/releases.exs

```diff
@@ -1,5 +1,10 @@
 import Config
 
+bullhorn_config =
```

Comment:
Just merge all the config. We can do that in this PR or the cd PR to come

Reply:
Done